Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBTransportDefaultConfig.m

```diff
@@ -105,16 +105,39 @@ - (instancetype)initWithAppKey:(NSString *)appKey
                  delegateQueue:(nullable NSOperationQueue *)delegateQueue
         forceForegroundSession:(BOOL)forceForegroundSession
      sharedContainerIdentifier:(nullable NSString *)sharedContainerIdentifier {
-  if (self = [super initWithAppKey:appKey
-                         appSecret:appSecret
-                    hostnameConfig:hostnameConfig
-                         userAgent:userAgent
-                        asMemberId:asMemberId
-                 additionalHeaders:additionalHeaders]) {
-    _delegateQueue = delegateQueue;
-    _forceForegroundSession = forceForegroundSession;
-    _sharedContainerIdentifier = sharedContainerIdentifier;
-  }
+    return [self initWithAppKey:appKey
+                      appSecret:appSecret
+                 hostnameConfig:hostnameConfig
+                    redirectURL:nil
+                      userAgent:userAgent
+                     asMemberId:asMemberId
+              additionalHeaders:additionalHeaders
+                  delegateQueue:delegateQueue
+         forceForegroundSession:forceForegroundSession
+      sharedContainerIdentifier:sharedContainerIdentifier];
+}
+
+- (instancetype)initWithAppKey:(NSString *)appKey
+                     appSecret:(nullable NSString *)appSecret
+                hostnameConfig:(nullable DBTransportBaseHostnameConfig *)hostnameConfig
+                   redirectURL:(nullable NSString *)redirectURL
```

Comment:
here too

Reply:
Oops, sorry for missing them. Removed all nullable from this .m file.